lib/dao.js

```diff
@@ -2334,6 +2335,12 @@ DataAccessObject.prototype.updateAttributes = function updateAttributes(data, op
     if (err) return cb(err);
     data = ctx.data;
 
+    if (strict) {
+      var keys = Object.keys(Model.definition.properties);
```

Comment:
We might ... what do you think @raymondfeng ? I haven't used `throw` or `validate` yet, can you explain? I don't believe these other settings are documented right now.